lib/csv_helper/loader.rb

```diff
@@ -73,5 +54,22 @@ def merge_options(options)
       options.reverse_merge(key_mapping: key_mapping, value_converters: value_converters)
              .reverse_merge(SMARTER_CSV_OPTIONS)
     end
+
+    # default validations are run during import to prevent bad data from going into table,
+    # this runs validations with a context of :load_csv
+    # validations with this context are ran now due either the validation needing data in the table being loaded into
+    # or the validation causes issues on save
+    def load_csv_validation(records, failed_instances, options)
```

Comment:
Can we rename this something more descriptive like `after_import_validations`?

Reply:
changed